github-action/entrypoint.sh

```diff
@@ -2,5 +2,8 @@
 set -e
 echo "::debug::\$cmd: $1"
 RESULT=$(eval "$1")
+RESULT="${RESULT//'%'/'%25'}"
+RESULT="${RESULT//$'\n'/'%0A'}"
+RESULT="${RESULT//$'\r'/'%0D'}"
```

Comment:
This escape logic comes from [a Github support case](https://github.community/t/set-output-truncates-multiline-strings/16852/3)

Reply:
bash black magic 👍🏼